src/validation.cpp

```diff
@@ -974,7 +974,7 @@ bool MemPoolAccept::Finalize(ATMPArgs& args, Workspace& ws)
     // Remove conflicting transactions from the mempool
     for (CTxMemPool::txiter it : allConflicting)
     {
-        LogPrint(BCLog::MEMPOOL, "replacing tx %s with %s for %s BTC additional fees, %d delta bytes\n",
+        LogPrint(BCLog::MEMPOOL, "replacing tx %s with %s for %s " + CURRENCY_UNIT + " additional fees, %d delta bytes\n",
```

Comment:
```suggestion
        LogPrint(BCLog::MEMPOOL, "replacing tx %s with %s for %s additional fees, %d delta bytes\n",
```

I think we can just remove the unit here

Reply:
Good point, I agree as this is just a log message and not really user visible.  I've removed it here.